src/main/webapp/css/style.css

```diff
@@ -22,3 +17,11 @@ h1 {
 .green {
   color: #0F9D58;
 }
+
+.logo {
+  text-decoration: none;
+}
+
+#___gcse_0 {
+  width: 60%;
```

Comment:
Adjusting to the parent width is nice, but you probably want to set a min-width and a max-width.